Resources/Prototypes/Reagents/gases.yml

```diff
@@ -35,6 +35,14 @@
         ratios:
           CarbonDioxide: 1.0
           Oxygen: -1.0
+    Poison:
+      effects:
+      - !type:HealthChange
+        damage:
+          types:
+            Cold: 2 #Liquid O2 is cold and will cause damage if injected.
+      - !type:AdjustTemperature
+        amount: -5000 #Since Liquid O2 is so cold, we reduce the temperature of anyone it is injected into.
```

Comment:
Does thist just kill them or

Reply:
Review media for effect from 15u. It's less potent than non-OD insuzine. It doesn't kill, but they'll definitely know they've just been exposed to something cold